doc/src/explore/sui-explorer.md

```diff
@@ -0,0 +1,120 @@
+
+The transaction details pages include the following tabs:
+ * **Details** - Provides additional details about the transaction.
+ * **Events** - Displays the events associated with the transaction.
+ * **Signatures** - Lists the signatures from validators for the transaction.
+
+The **Details** tab includes the following fields:
+ * **Package Details** - Displayed only for Call transactions. Displays the Package ID associated with the transaction and the following fields:
+     * Module - The module used for the transaction.
+     * Function - The Function called for the transaction.
+     * Argument - Any arguments included with the function.
+ * **Updated** - The object ID for the object the transaction updated. Click the ID to view the [Object details](#object-details-page) page for more details.
+ * **Created** - The object ID for the object this transaction created. Click the ID to view the [Object details](#object-details-page) page for more details.
+ * **Amount** - The number and type of coins transferred for the transaction.
+ * **Sender** - Displayed only for Publish transactions. The address of the sender of the transaction.
+ * **Sender & Recipients** - The addresses associated with the transaction. The first value is the sender's address, and the address next to the green checkmark is the recipient's address. When there are multiple recipients, the field includes multiple addresses.
+ * **Modules** - Shows the module code used to create and execute the transaction.
+ * **Gas and storage fees** - Details about the gas and fees for the transaction. 
+The value for Gas Payment is the object ID for the coin object used for the transaction.
+ * **Gas Fees** - The number of gas units used for the transaction. 
+ * **Gas Budget** - The maximum number of gas units allowed for the transaction.
+
+The **Events** tab lists the events the transaction generated and the details about each event. TransferSui transactions do not include events.
+
+The **Signatures** tab includes the following fields:
+ * **Transaction Signatures** - The signature or signatures for the transaction.
+ * **Validator Signatures** - The signatures from the validators that validated the transaction.
+
+## Object details page
+When you click on an object ID displayed on a transaction details page it opens a page that displays the details for the object, such as the transactions associated with the object. The page includes the following details:
+
+ * **Description** includes the following fields:
+     * **Type** - The type of the object, such as coin.
+     * **Object ID** - The ID of the object. 	
+     * **Last Transaction ID** - 	The ID of the most recent transaction associated with the object.
+     * **Version** - The version of the object. 
+     * **Owner** - The address of the owner of the object.
+ * **Properties** - Displays details such as the coin balance for the object.
```

Comment:
Might be better to strip the verbs for consistency. There are 20 instances of "Display" on this page now...